tests/db/test_outbox.py

```
@@ -0,0 +1,94 @@
+                "UPDATE outbox_messages SET claimed_at = CURRENT_TIMESTAMP - interval '10 minutes' WHERE id = $1",
+                mid,
+            )
+            reclaimed = _j(await conn.fetchval("SELECT claim_pending_outbox(50, 60)"))
+            assert [c["id"] for c in reclaimed] == [str(mid)]
+        finally:
+            await tr.rollback()
+
+
+async def test_empty_message_rejected(db_pool):
+    async with db_pool.acquire() as conn:
+        with pytest.raises(Exception):
```

Ruff (0.15.20)

[warning] Do not assert blind exception: `Exception`

(B017)